plugins/imagebase/plugin.js

```diff
@@ -44,8 +54,56 @@
 		return CKEDITOR.plugins.link.parseLinkAttributes( widget.editor, widget.parts.link );
 	}
 
-	var featuresDefinitions = {
-		link: {
+	function getLinkFeature() {
+		function isNotLinkableOrIsLink( widget, dialog ) {
```

Comment:
This function name is misleading. I'd rather leave this condition in original form, without introducing such helper function.